<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QWidget" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>739</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(E:/uiProject/help.png);
border-radius: 35px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>740</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(E:/uiProject/c.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>740</y>
     <width>101</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(E:/uiProject/c.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="infoButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>739</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(E:/uiProject/info.png);
border-radius: 35px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="classFour">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>520</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(E:/uiProject/classOne.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="achiveButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>740</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(E:/uiProject/achive.png);
border-radius: 35px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>740</y>
     <width>101</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(E:/uiProject/c.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>740</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(E:/uiProject/c.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>646</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(E://uiProject//happy.png);
background-position:center;
background-origin:content;
background-repeat:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="classThree">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>390</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(E:/uiProject/classOne.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="classTwo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>400</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(E:/uiProject/classOne.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="classOne">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(E:/uiProject/classOne.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="letterButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>740</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(E:/uiProject/letter.png);
border-radius: 35px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="game1Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>game1Btn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="game2Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>game2Btn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="game3Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>game3Btn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="game4Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>game4Btn</string>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>helpButton</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>classFour</zorder>
  <zorder>achiveButton</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>classThree</zorder>
  <zorder>classTwo</zorder>
  <zorder>classOne</zorder>
  <zorder>letterButton</zorder>
  <zorder>infoButton</zorder>
  <zorder>game1Btn</zorder>
  <zorder>game2Btn</zorder>
  <zorder>game3Btn</zorder>
  <zorder>game4Btn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
